packages/core/src/shared/clients/s3Client.ts

```diff
@@ -450,7 +451,12 @@ export class DefaultS3Client {
                 Bucket: bucket.name,
                 Delimiter: DEFAULT_DELIMITER,
                 MaxKeys: request.maxResults ?? DEFAULT_MAX_KEYS,
-                Prefix: request.folderPath,
+                /**
+                 * Set '' as the default prefix to ensure that the bucket's content will be displayed 
```

Comment:
JSDoc has a property `@default`. We can probably use that here. This way, IDEs can directly display the default values. 

```
/**
* Set '' as the default prefix to ensure that the bucket's content will be displayed
* when the user has at least list access to the root of the bucket.
* https://github.com/aws/aws-toolkit-vscode/issues/4643
* @default '' 
```

Reply:
Adjusted on 468270c.